<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1569</width>
    <height>880</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(2,62,94,255);
color: rgba(26,143,195,255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="projectName">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>89</green>
          <blue>141</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="brand">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="model">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);

color : white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serialNo">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantity">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>260</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Product Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Brand</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Model:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>
Serial number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Piece</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Purchase Date
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="description">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Explanation:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>560</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>90</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 20pt &quot;Segoe UI&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Product Information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="readProducts">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>130</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>550</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>550</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>90</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
color : white;</string>
    </property>
    <property name="text">
     <string>Show Products</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>product.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>700</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(34,89,141,255);
font: 12pt &quot;Segoe UI&quot;;
color : white;</string>
    </property>
    <property name="text">
     <string>                            Double click on the product you want to delete!!!</string>
    </property>
   </widget>
   <zorder>readProducts</zorder>
   <zorder>projectName</zorder>
   <zorder>brand</zorder>
   <zorder>model</zorder>
   <zorder>serialNo</zorder>
   <zorder>quantity</zorder>
   <zorder>price</zorder>
   <zorder>dateEdit</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>description</zorder>
   <zorder>label_8</zorder>
   <zorder>addButton</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_pic</zorder>
   <zorder>label_10</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1569</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
